courses/machine_learning/deepdive2/recommendation_systems/solutions/content_based_by_hand.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<p>Add period to end.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/691/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Ack</p>